src/Integration.UnitTests/ProfileConflicts/RuleSetConflictsControllerTests.cs

```diff
@@ -0,0 +1,245 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="RuleSetConflictsControllerTests.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using Microsoft.VisualStudio.CodeAnalysis.RuleSets;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarLint.VisualStudio.Integration.ProfileConflicts;
+using SonarLint.VisualStudio.Integration.TeamExplorer;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Windows.Input;
+using System.Windows.Threading;
+
+namespace SonarLint.VisualStudio.Integration.UnitTests
+{
+    [TestClass]
+    public class RuleSetConflictsControllerTests
+    {
+        private ConfigurableHost host;
+        private ConfigurableServiceProvider serviceProvider;
+        private ConfigurableVsGeneralOutputWindowPane outputWindow;
+        private ConfigurableRuleSetInspector ruleSetInspector;
+        private ConfigurableSourceControlledFileSystem sccFS;
+        private ConfigurableRuleSetSerializer rsSerializer;
+
+        [TestInitialize]
+        public void TestInit()
+        {
+            this.serviceProvider = new ConfigurableServiceProvider();
+            this.outputWindow = new ConfigurableVsGeneralOutputWindowPane();
+            this.serviceProvider.RegisterService(typeof(SVsGeneralOutputWindowPane), this.outputWindow);
+            this.host = new ConfigurableHost(this.serviceProvider, Dispatcher.CurrentDispatcher);
+        }
+
+        [TestCleanup]
+        public void TestCleanup()
+        {
+            this.ruleSetInspector = null;
+            this.sccFS = null;
+            this.rsSerializer = null;
```

Comment:
Why not null them all out in _TestInit_? That would make explicit what the expected state at the start of each test is.